signcrypt_test.go

```diff
@@ -192,9 +193,9 @@ func TestSigncryptionPacketSwappingWithinMessage(t *testing.T) {
 	headerLen := getHeaderLen(t, sealed)
 	packetLen := getPayloadPacketLen(encryptionBlockSize)
 	packet2Start := headerLen + packetLen
-	emptyPacketLen := getPayloadPacketLen(0)
+	// TODO: Rem empty packet ref.
```

Comment:
Just the variables referring to the `emptyPacket` (which are now empty slices, etc.). Removed.